src/components/hooks/usePublicationStatus.tsx

```diff
@@ -83,6 +84,8 @@ const computePublicationStatus = ({
   switch (true) {
     case isPublishedChildren({ item, publishedEntry }):
       return PublicationStatus.PublishedChildren;
+    case itemTypeNotAllowedToBePublished(item.type):
```

Comment:
Same remark as in the backend. You can create your own function for the frontend.

Reply:
As I wrote in the backend PR, for me it is not an issue to use a function from the SDK, maybe we can discuss in the backend's PR ?